lib/events/auditlog.go

```diff
@@ -109,7 +109,7 @@ func init() {
 // sessions. It implements IAuditLog
 type AuditLog struct {
 	sync.Mutex
-	*log.Entry
+	log.FieldLogger
```

Comment:
`logrus.Entry` in v1.6.0 embeds a `context.Context` which causes disambiguation errors here and below.

Reply:
Could you refactor this to not embed the logger (name this field `log`)?
Otherwise we pollute the type's API for a few saved characters.